tests/archives/conftest.py

```diff
@@ -140,17 +154,32 @@ def get_archive_data(name, dtype=np.float64):
             dtype=dtype)
         grid_indices = (6, 11)
         int_index = 131
-
-    archive_with_elite.add_single(solution, objective, measures)
+    elif name == "UnstructuredArchive":
+        cells = 0
+        archive = UnstructuredArchive(solution_dim=len(solution),
+                                      measure_dim=2,
+                                      k_neighbors=5,
+                                      sparsity_threshold=1.0,
+                                      dtype=dtype)
+
+        archive_with_elite = UnstructuredArchive(solution_dim=len(solution),
+                                                 measure_dim=2,
+                                                 k_neighbors=5,
+                                                 sparsity_threshold=1.0,
+                                                 dtype=dtype)
+        grid_indices = (0,)
+        int_index = 0
+
+    archive_with_elite.add_single(solution, objective, measures)  # pylint: disable=E
 
     return ArchiveFixtureData(
-        archive,
+        archive,  # pylint: disable=E
```

Comment:
This must be an accident - I always mean to put the full code. In this case it was [E0606](https://pylint.readthedocs.io/en/stable/user_guide/messages/error/possibly-used-before-assignment.html)

That was being flagged by pre-commit/pylint because `archive` is only defined inside conditional statements and there is no `else` clause defining it. I ignored it because this was the same structure/behavior you had before, and if `archive` is not defined the test would fail anyway.